utils/rpms/daos.spec

```
@@ -328,6 +329,8 @@ mv test.cov-build %{buildroot}/usr/lib/daos/TESTING/ftest/test.cov
 %endif
 mkdir -p %{buildroot}/%{_sysconfdir}/ld.so.conf.d/
 echo "%{_libdir}/daos_srv" > %{buildroot}/%{_sysconfdir}/ld.so.conf.d/daos.conf
+mkdir -p %{buildroot}/%{_sysconfdir}/sysctl.d/
+install -m 644 utils/rpms/sysctl.d/%{sysctl_script_name} %{buildroot}/%{_sysconfdir}/sysctl.d
```

This location is actually defined by the systemd package:

```
$ rpm --eval '%{_sysctldir}'
/usr/lib/sysctl.d
```

which is a BR: for daos so we should use it.  But you also cannot reference files outside of the RPM the way you do with `utils/rpms/sysctl.d`.  That location will not exist on another site where somebody tries to rebuild the SRPM.  You need to include the file as a `$SOURCE` file (i.e. `Source2: %{sysctl_script_name}` on line 25) and reference it that way:

```suggestion
mkdir -p %{buildroot}/%{_sysctldir}
install -m 644  %{SOURCE2} %{buildroot}/%{_sysctldir}
```

and indeed `/usr/lib/sysctl.d` is the correct location for an RPM to install it's configurations.  `/etc/` files/dirs are considered to be for the sysadmin to apply local configurations.